### ISSUE TITLE
tests in tools/ci_connector_ops are not run in CI 

### ISSUE THREAD

**Issue by erohmensing** [airbytehq/airbyte#28494]

**Comment by alafanechere** [comment]
Relates to https://github.com/airbytehq/airbyte/issues/28426 which is specific to pipelines unit test ran.

@erohmensing would you mind splitting this issue for each package in ci_connector_ops that will end up in `airbyte-ci`?
```
airbyte-ci/connectors/common_utils
airbyte-ci/connectors/pipelines
airbyte-ci/connectors/credentials
```

**Comment by bnchrch** [comment]
Currently moving these tests in #25710

**Comment by erohmensing** [comment]
@bnchrch to be clear your comment only applies to `airbyte-ci/connectors/pipelines` right? 

I can create new issues like https://github.com/airbytehq/airbyte/issues/28426 for 
`airbyte-ci/connectors/common_utils`
`airbyte-ci/connectors/credentials`
and add them to the CI/CD stability epic but not the pipeline burndown epic. make sense?

**Comment by bnchrch** [comment]
@erohmensing Hmm nope those are also being moved in this big hairy PR

https://github.com/airbytehq/airbyte/pull/27957/files

**Comment by erohmensing** [comment]
Huh, cool! @alafanechere do you still want me to make separate issues then?

**Comment by alafanechere** [comment]
I don't think this issue belongs to the "Pipeline stability task force" epic as running the pipelines unit test is covered by #28426 .
https://github.com/airbytehq/airbyte/pull/28857 created an easily extendable workflow to which we could add test run for other `airbyte-ci` python packages.

We could run tests for :
airbyte-ci/connectors/common_utils
airbyte-ci/connectors/credentials

in this action, given we currently have healthy test suites.
